api/schema/XmlSchemaCache.inc

```diff
@@ -46,10 +51,10 @@ class XmlSchemaCache {
    * @return bool
    *   TRUE if the cache was created or already exists; FALSE otherwise.
    */
-  public static function Create() {
-    if (!self::Exists()) {
+  public static function create() {
+    if (!static::Exists()) {
       // Clear will initialize the table and files location.
-      return self::Clear();
+      return static::Clear();
```

Comment:
Should be updated: https://github.com/discoverygarden/xml_forms/pull/3/files#diff-2dfdc52537db07398e791f49e643b03dR69